release/src/slack.ts

```diff
@@ -182,58 +224,124 @@ function githubRunLink(
   );
 }
 
-export async function sendReleaseMessage({
+export async function sendPreReleaseMessage({
   github,
   owner,
   repo,
-  stage,
   version,
   runId,
   releaseSha,
+  channelName,
+  userName,
 }: ReleaseProps & {
-  stage: BuildStage;
   version: string;
   runId: string;
   releaseSha: string;
+  channelName: string,
+  userName: string,
 }) {
-
   const title = getReleaseTitle(version);
   const space = "\n";
```

Comment:
Surely this should be called `linebreak`?

Reply:
😅